otwcode/otwarchive
# AO3-5789 Update Database Cleaner and test-only gems

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-5789

## Purpose

What does this PR do?

This PR updates the test-only gems that are not locked to a specific version.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended? (If you have access, please copy this into the JIRA ticket for them!)

There is nothing to test, since I've only included gems that are used exclusively by the automated test suite.

## References

Are there any other relevant issues / PRs / mailing lists discussions related to this?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

Enigel

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

she/her
